tests/run-make/foreign-double-unwind/Makefile

```diff
@@ -1,12 +0,0 @@
-# ignore-cross-compile
-# needs-unwind
-include ../tools.mk
-
-all: foo
-	$(call RUN,foo) | $(CGREP) -v unreachable
-
-foo: foo.rs $(call NATIVE_STATICLIB,foo)
-	$(RUSTC) $< -lfoo $(EXTRARSCXXFLAGS)
```

Comment:
Question: is the exit code of this checked at all? The error message could be intentional (2 exceptions thrown in the C++ code - that's what the test is trying to check)